### PR TITLE
feat: init

### PR DESCRIPTION
**Pull request by atian25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,98 @@
...
+
+- 配置 `npm scripts`：
+
+```json
+{
+  "name": "your-project",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "mocha",
+    "ci": "c8 npm test"
```
Comment (hyj1991) [inline README.md:15]: `c8` 现在可以完全替代 `nyc` 了么

Comment (atian25) [reply README.md:15]: 我觉得可以，直接用 v8 的覆盖率多香。它这里的 c8 其实只是设置了一个环境变量，然后等 npm test 执行完后，再根据 v8 导出的内容去生成报告。

印象中就是一些进程退出的 edge case 可能没有 nyc 覆盖全

Comment (hyj1991) [reply README.md:15]: 有空我试试对 `worker_threads` 执行的代码是否可以扫到，`nyc` 似乎无法判断 `worker_threads` 执行的代码

Comment (atian25) [reply README.md:15]: 如果扫不到的话，就找 秋怡～ 反正 v8 这个特性是她在 Node 里面支持的
